src/Module/module.css

```diff
@@ -17,48 +17,44 @@
 {
     & > .header
     {
+        background: var( --GS-11 );
         cursor:     pointer;
-        background: var(  --GS-11  );
-        transition: height 200ms ease-in-out, background-color 200ms ease-in-out;
     }
 }
 
 .collapsed
 {
-    padding-bottom:  0;
-
-    & > .header
+    & .header
     {
-        padding-bottom:     3rem;
-        margin-bottom:      0;
+        background:         transparent;
 
-        background: var(  --GS-11  );
-        transition: height 200ms ease-in-out, background-color 200ms ease-in-out;
-        height:     100%;
+        &:hover
+        {
+            background: var( --GS-11 );
+        }
     }
 
-    &:hover > .header
-    {
-        background: transparent;
-        transition: height 200ms ease-in-out, background-color 200ms ease-in-out;
-    }
     .content
     {
+        background: var( --GS-11 );
         display:    none;
     }
+
 }
 
 .moduleError
 {
-    & > .header
+    border-left: 0.5rem solid var( --AC-1 );
+
+    & > .header, .content
```

Comment:
This _sort of_ works, but it’s not quite right...

The rule:
```
& > .header, .content
{
    padding-left: 2.5rem;
}
```

compiles to:

```
.module__moduleError > .module__header, 
.module__moduleError .module__content
{
    padding-left: 2.5rem;
}
```

But... what we really want is:
```
.module__moduleError > .module__header, 
.module__moduleError > .module__content
{
    padding-left: 2.5rem;
}
```

(Note the second `>`)

This is what we need to do:
```
& > .header, 
& > .content
{
    padding-left: 2.5rem;
}
```






Why does this matter? If you have something like this:

```
<Module hasModuleError>
    <Module>...</Module>
    <Module>...</Module>
</Module>
```

then we’d be applying the `padding-left: 2.5rem` not just to the `.content` div of the outer module, but also to the `.content` div of _every_ module nested inside it!